### ISSUE TITLE
Buildifier does not support f-strings

### ISSUE THREAD

**Issue by cbarrete** [bazelbuild/buildtools#1278]
Buck2 supports f-string (although [Bazel does not](https://github.com/bazelbuild/starlark/issues/91), at least yet), so it would be nice if `buildifier` did not consider them invalid syntax.

Currently, running it over:

```starlark
name = "Cédric"
print(f"Hello, {name}!")
```

gives

```
> buildifier asdf.bzl
asdf.bzl:2:24: syntax error near "Hello, {name}!"
asdf.bzl # reformat
```

It seems like this could be implemented quite easily similarly to [raw strings](https://github.com/bazelbuild/buildtools/blob/main/build/lex.go#L547), is this correct?

**Comment by vladmos** [comment]
That's correct, however the linter check for unused variables also need to be updated (e.g. the following code shoudln't issue warnings):

    def f():
        foo = 'bar'
        return f'{bar}'

**Comment by aryandutt** [comment]
It looks like a good first issue, can I work on it?

**Comment by cbarrete** [comment]
This is still an issue.

**Comment by nmedveditsko-nvidia** [comment]
This has been addressed in https://github.com/bazelbuild/buildtools/pull/1465.  Please share what you think.

**Comment by cbarrete** [comment]
Nice, thank you!
I will try to test this when I have a bit of time.

**Comment by nmedveditsko-nvidia** [comment]
@cbarrete  - did you have a chance to take a look?
